pkgs/os-specific/darwin/darwin-stubs/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation {
+  pname = "darwin-stubs";
+  version = "10.12";
+
+  src = fetchFromGitHub {
+    owner = "thefloweringash";
+    repo = "darwin-stubs";
```

Comment:
if this is directly to support nixpkgs it would probably be a good idea to transfer the repo here? And also have darwin maintainers access to it

Reply:
I would definitely like to do this. I included it in the "Prerequisites" section, which isn't quite panning out as I expected.

Reply:
I think @domenkozar is one of the people with enough permissions to do that.

Reply:
If someone PM's him on IRC it should be pretty easy to have done

Reply:
@thefloweringash if you add me to that repo I can transfer it.